tool/gravity/cli/install.go

```diff
@@ -793,10 +793,14 @@ var InterruptSignals = signals.WithSignals(
 func NewInstallerConnectStrategy(env *localenv.LocalEnvironment, config InstallConfig, commandArgs cli.CommandArgs) (strategy installerclient.ConnectStrategy, err error) {
 	commandArgs.FlagsToAdd = append(commandArgs.FlagsToAdd,
 		cli.NewFlag("token", config.Token),
-		cli.NewBoolFlag("selinux", config.SELinux),
 		cli.NewBoolFlag("from-service", true),
 		cli.NewArg("path", config.StateDir),
 	)
+	if config.Mode != constants.InstallModeInteractive {
+		commandArgs.FlagsToAdd = append(commandArgs.FlagsToAdd,
```

Comment:
I'm not too familiar with the code yet, but could there be any problems if the `selinux` flag is not appended to `FlagsToAdd` but appended to `FlagsToRemove`?

Reply:
The way it works is `FlagsToRemove` are handled first and they do not require the flag to actually be present on command line. After that, the code will update the command line with the flags given with `FlagsToAdd` so it should not break anything.